build/bin/bridge-over-nic

```diff
@@ -0,0 +1,49 @@
+#!/usr/bin/env bash
+
+set -xe
+
+bridge_name=$1
+nic_name=$2
+vlan_range_min=$3
+vlan_range_max=$4
+
+VLAN_RANGE="${vlan_range_min}-${vlan_range_max}"
```

Comment:
vlan_range_min/max should be mandatory ?
if so maybe worth to add a check like the ones below ?

Reply:
good point, thanks